concent_api/core/tests/test_core_views.py

```diff
@@ -39,21 +40,19 @@ class CoreViewSendTest(TestCase):
     @freeze_time("2017-11-17 10:00:00")
```

Comment:
You should put `CONCENT_MESSAGING_TIME` in `override_settings` to ensure that tests still work even if we change the default in settings.